content/contents+de.lr

```diff
@@ -82,98 +86,67 @@ resources_materials_heading: Materialien
 ---
 resources_materials_text:
 
-Erzeuge doch ein Logo mit diesem [Logo-Generator](http://odd.structure-and-representation.com/)!
+Erzeuge ein Logo für deiner Stadt mit diesem [Logo-Generator](http://odd.structure-and-representation.com/)!
 ---
 resources_ideas_heading: Ideen und Datenquellen
 ---
 resources_ideas_intro:
 
-Hier ein paar Ideen was du alles am Open Data Day tun könntest:
+Hallo da offener Datenenthusiast(in)! Du hast gerade gesehen, dass der Tag der offenen Daten am Samstag, den 7. März 2020, stattfindet und bist daran interessiert, eine Veranstaltung in deine Nähe zu veranstalten? Die einzige Herausforderung ist es jetzt, die Idee zu finden, die den richtigen Ort trifft. Unsere Freunde aus der Open Data Community haben ein paar Datenquellen, die es wert sind, dass du dich informierst.
+
+Besuche die [CKAN-Website](https://ckan.org/about/instances/), um alle Organisationen auf der ganzen Welt zu entdecken, die CKAN zur Veröffentlichung ihrer Daten nutzen.
+
+Verwende [data.world](https://data.world), um Daten aus vielen Quellen hochzuladen oder zu finden und alle Aspekte eines Projekts - einschließlich Daten, Notizbücher, Analysen und Diskussionen - in einem einzigen Arbeitsbereich zu organisieren.
```

Comment:
I don't think there's a good German translation for computational notebooks, so I'd just stick to the english term and capitalize it -> "einschließlich Daten, Notebooks, Analysen und Diskussionen"